tools/cli/src/mentat_cli/input.rs

```diff
@@ -80,7 +86,19 @@ impl InputReader {
 
         self.add_history(&line);
 
-        let cmd = try!(command(&self.buffer));
+        let cmd = match in_process_cmd {
+            Some(Command::Query(args)) => {
+                Command::Query(args + " " + &line)
+            },
+            Some(Command::Transact(args)) => {
+                Command::Transact(args + " " + &line)
+            },
+            _ => {
```

Comment:
Prefer to explicitly match `Some(...)` and `None` here, so that the compiler will tell you when you need to handle cases.  I feel like there's an abstraction around continuable commands vs. non-continuable commands missing here, but it's your choice how to express this.